### ISSUE TITLE
Data issue: power_trajectory shutdown must be above power minimum

### ISSUE THREAD

**Issue by kdayday** [NREL-SIIP/ExtremeSolarTexas#14]
This is an analogous problem to the startup trajectories needing to be above the minimum power level.

Some units have a shutdown power trajectory of 0.0 (implying I think that they can shut down entirely in one go). However, that makes the MultiStart shutdown equation infeasible (RHS of pglib equation 18). The shutdown power trajectory must be at least the minimum power level to make the RHS positive.

A shutdown trajectory (SD) of the minimum power level implies the unit must come to minimum before turning off; for units that can turn off immediately, I think SD should equal the maximum power rating, which makes the coefficient of the shutdown status w_g[g, t+1] equal 0 -- because it should be irrelevant if you can shut down at any time.

**Comment by kdayday** [comment]
I looked into these plants a bit more. I updated CC_CT and GT shutdown to 1, which will fix those two categories. The remaining issue is that 52 of the 88 CC plants have a minimum:maximum rating ratio greater than the 0.5 used below (ranges from 0.5 up to 0.92), so those plants will need to have the shutdown overwritten with the minimum rating. 

```
power_trajectory = Dict(
    ("ST", "LIG") => (startup = 0.5, shutdown = 0.5),
    ("ST", "SUB") => (startup = 0.5, shutdown = 0.5),
    ("CC_CA", "NG") => (startup = 0.5, shutdown = 0.5),
    ("CC_CT", "NG") => (startup = 1.0, shutdown = 1.0),
    ("GT", "NG") => (startup = 1.0, shutdown = 1.0),
    ("ST", "NG") => (startup = 0.8, shutdown = 0.5),
)
```

**Comment by jd-lara** [comment]
closed via b099cedface250d43cdcd5176a213bd55d5ef1d0 

@kdayday can you check the commit to see if it is what you needed please?
